Benchmark Tooling (#389)
Modalities/modalities: tutorials/scaling_up/README.md

The config file hash is also used as the name for the experiment config. 

```txt
├── README.md
├── analysis
│   └── throughput_analysis.ipynb
├── configs
│   ├── sweep_8B_fsdp2.yaml
│   └── sweep_config.yaml
├── data
│   └── lorem_ipsum_long.pbin
├── experiments
│   ├── 2025-07-16__17-28-03_970fedec
│   │   ├── 2
│   │   │   ├── abcd1234_2025-07-16__17-30-00
│   │   │   │   ├── abcd1234.yaml
│   │   │   │   ├── abcd1234.yaml.resolved
│   │   │   │   └── evaluation_results.jsonl
│   │   │   ├── abcd1234_2025-07-24__21-23-04
│   │   │   │   ├── abcd1234.yaml
│   │   │   │   ├── abcd1234.yaml.resolved
│   │   │   │   └── evaluation_results.jsonl
│   │   │   └── abcd1234_2025-07-24__21-23-37
│   │   │       ├── abcd1234.yaml
│   │   │       ├── abcd1234.yaml.resolved
│   │   │       └── evaluation_results.jsonl
│   │   ├── 4
│   │   │   ├── abcd1234_2025-07-16__17-30-00
│   │   │   │   ├── ...
│   │   │   ├── ...
│   │   └── ...
└── scripts
    ├── create_sweep_configs.sh
    ├── hpc
    │   └── leonardo
    │       ├── job.sbatch
    │       └── submit_sweep.sh
    └── single_node
        └── run_single_node.sh
```

## Running the Experiments

> To run the experiments, you can either run them on a single node or on a cluster.

Reviewer: ```suggestion
The experiments can be run either on a single node or on a cluster.
```
